### ISSUE TITLE
How to specify the embedding or softmax dimension?

### ISSUE THREAD

**Issue by anglil** [tensorflow/tensor2tensor#231]
which parameter in common_hparams correspond to the dimension of the embedding or softmax layer?

**Comment by lukaszkaiser** [comment]
Currently, our embeddings are the same size as `hidden_size`, we don't have a parameter only for embedding/softmax size (as we probably should).

**Comment by anglil** [comment]
Thanks for the info!
